README.md

```diff
@@ -1,9 +1,17 @@
-# ffmpeg-binaries
-A simple way to install FFmpeg globally or for an individual project.  
-Supports Windows (ia32, x64), Linux (ia32, x64, arm, arm64) and macOS (x64)
+# node-ffmpeg-binaries
+A platform indepindependent installer of [FFmpeg](https://ffmpeg.org/)
+
+
+Supported architectures:
+Linux | Windows | Mac
+----- | ------- | ---
+x64   | x64     | x64
+ia32  | ia32    |
+arm   |         |
+arm64 |         |
 
 ## API
-`ffmpeg-binaries` exports a string containing the path to the FFmpeg executable.
+`node-ffmpeg-binaries` exports a string containing the path to the FFmpeg executable.
```

Comment:
On npm this is listed as `ffmpeg-binaries`, the repo is just prefixed with `node-` to show it's a node package

Reply:
Yes, as it should, the readme should always match the repo name so you know its for node.